ansible/roles/schulcloud-server-provisioning/templates/schulconnex-provisioning-configmap.yml.j2

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ NAMESPACE }}
+  name: schulconnex-provisioning-configmap
+  labels:
+    app: schulconnex-provisioning
+data:
+  NODE_OPTIONS: "--max-old-space-size=2048"
```

Comment:
Why 2048 if you have only 2GB Memory, what about other things what need memory.
Value shoud be like 1536, 1664 or 1792